### PR TITLE
Updating Design System package

### PR DESCRIPTION
**Pull request by amirpourpanahi**
### Description

List of proposed changes:

- Updating Design System package
- Changing DS Header props

### CODE REVIEW

File: locales/en.js [hunk 1]
```diff
@@ -313,6 +313,13 @@ export default {
   learnMoreAbout: 'Learn more about ',
   applyFor: 'Apply for',
 
+  // Header URLs
+  dashboardPath: '/dashboard',
+  craPath: '/',
```
Comment (s-laugh) [inline locales/en.js:318]:
```suggestion
  craPath: 'https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/account-individuals.html',
```

File: locales/fr.js [hunk 2]
```diff
@@ -305,6 +305,13 @@ export default {
   learnMoreAbout: '(FR)Learn more about ',
   applyFor: '(FR)Apply for',
 
+  // Header URLs
+  dashboardPath: '/dashboard',
+  craPath: '/',
```
Comment (s-laugh) [inline locales/fr.js:310]:
```suggestion
  craPath: 'https://www.canada.ca/fr/agence-revenu/services/services-electroniques/services-electroniques-particuliers/dossier-particuliers.html',
```

File: components/molecules/DSHeader.js [hunk 3]
```diff
@@ -4,41 +4,37 @@ import fr from '../../locales/fr'
...
-            link: t.url_serviceCanada,
-            text: t.serviceCanada,
-          },
-          {
-            link: t.url_myBenefitsAndServices,
-            text: t.myBenefitsAndServices,
-          },
-        ],
+      id="header"
+      lang={t}
```
Comment (s-laugh) [inline components/molecules/DSHeader.js:10]: We shouldn't pass our language file, just 'en' or 'fr' depending on the language.
```suggestion
      lang={props.locale}
```

**Review by s-laugh — CHANGES_REQUESTED** [review]
just a few tweaks to make

File: components/molecules/DSHeader.js (hunk 3 above)
Comment (amirpourpanahi) [reply components/molecules/DSHeader.js:10]: Done, thank you!

**Review by s-laugh — APPROVED** [review]
Great!

**Review by nabil386 — APPROVED** [review]

**Pull request merged**